jenkins/pipelines/ci/tidb/tidb_ghpr_unit_test.groovy

```diff
@@ -261,12 +261,18 @@ try {
                                 rm -rf /tmp/tidb
                                 set -e
                                 export log_level=info
-                                time GORACE="history_size=7" ${goTestEnv} go test -v -vet=off -p 5 -timeout 20m -race \$(cat packages_race_${chunk_suffix}) ${extraArgs} # > test.log
+                                time GORACE="history_size=7" ${goTestEnv} go test -v -vet=off -p 5 -timeout 20m -race \$(cat packages_race_${chunk_suffix}) ${extraArgs} | tee test.log
                                 """
                                 }
                             }catch (err) {
+                                println 
+                                sh """
+                                    cat test.log | cut -d" " -f2- | grep -Ev "^\\[[[:digit:]]{4}(/[[:digit:]]{2}){2}" | grep -A 30 "\\-------" | grep -A 29 "^FAIL:"
+                                """
+
                                 throw err
                             }finally {
+                                rm test.log
```

Comment:
why rm

Reply:
Because the log is already sent to stdout and just is teed to the `test.log` for the err handling to grep.